### DISCUSSION TITLE
Model severely underestimates the data but indicates a reasonable reduced chi-squared value.

### DISCUSSION THREAD

**Discussion by IsmailEissa** [mboquien/cigale#73] [Q&A]
So I realized for a certain batch in my data, there is an issue of producing a reasonable reduced chi-squared value but if you look at the actual SED fitting, the chi-squared value does not really add up with the SED model (check bad fit +good chi-squared fit). I attached two other models to show that the batch still produced good SEDs, and also bad SEDs that were easily noticed because of their high chi-squared value.  I also, added the prompt I get when fitting those objects.  Thanks in advance.
![SED_prompt](https://github.com/mboquien/cigale/assets/61694192/efa617ce-ce3f-41a6-90e9-72e293aaf8b0)

[good fit+good chi-squared.pdf](https://github.com/mboquien/cigale/files/13603539/good.fit%2Bgood.chi-squared.pdf)
[bad fit+ good chi-squared.pdf](https://github.com/mboquien/cigale/files/13603543/bad.fit%2B.good.chi-squared.pdf)
[bad fit+ bad chi-squared.pdf](https://github.com/mboquien/cigale/files/13603545/bad.fit%2B.bad.chi-squared.pdf)

**Comment by Guang91** [comment]
The 2nd and 3rd pdf files are weird. I don't know why all the data are systematically higher than the model. @mboquien  any idea what's wong?

**Comment by mboquien** [comment]
This is quite curious. What version are you using? Also, would you mind providing a minimal example (input flux file+`pcigale.ini`+`pcigale.ini.spec`). Thanks!

**Reply by IsmailEissa** [reply]
Im using the 2022.1 version , here is a sample of the input file
[Showcasing_NGP.txt](https://github.com/mboquien/cigale/files/13886402/Showcasing_NGP.txt) , pcigale.ini ( 
[NG_262_.ini.txt](https://github.com/mboquien/cigale/files/13886407/NG_262_.ini.txt) and pcigale.ini.spec (
[NG_262_.ini.spec.txt](https://github.com/mboquien/cigale/files/13886413/NG_262_.ini.spec.txt)
) 

I tested again without the radio modules and it provided better fits.
First galaxy has a good fit: 
[124522.98+315219.4_best_model.pdf](https://github.com/mboquien/cigale/files/13886474/124522.98%2B315219.4_best_model.pdf)
First galaxy was also ran without the radio and it also fit well, but I just didnt attach it
Second galaxy has a bad fit: 
[(With Radio)124938.21+321111.5_best_model.pdf](https://github.com/mboquien/cigale/files/13886479/With.Radio.124938.21%2B321111.5_best_model.pdf)
So I tried fitting without the radio module and got a better one
[(Without Radio)124938.21+321111.5_best_model.pdf](https://github.com/mboquien/cigale/files/13886482/Without.Radio.124938.21%2B321111.5_best_model.pdf)

**Reply by mboquien** [reply]
Thanks. Could you also send the user-defined filters? By any chance, have you managed to reproduce the issue with a somewhat smaller parameter space? Another question, when you look at the Bayesian estimates of the flux, are they correct? That is, only the best-fit values are problematic?

If you can the chance, could you also try with https://gitlab.lam.fr/cigale/cigale/-/archive/develop/cigale-develop.tar.gz ?

**Reply by IsmailEissa** [reply]
Hey, the filters are made: 
[0p2to12.txt](https://github.com/mboquien/cigale/files/13902257/0p2to12.txt) and 
[LoLSS_LOFAR.txt](https://github.com/mboquien/cigale/files/13902270/LoLSS_LOFAR.txt). I just tested yesterday to remove the excess values in the SKIRTOR model( the ones in the pcigale.ini I attached) and plug in the default one again E(B-V)=0.03, and it worked with no faulty chi-squares like before, but the AGN emission doesnt dominate anymore in some cases ( which shouldnt be the case for type-1 QSOs), so that solves it? But I am not sure why would it work for other fields with no problem whatsoever, but certain fields, it did the aforementioned problem. I should also mention, that the fields that had no problem, were a much smaller (approx. 20 objects ) sample size than the ones that had (approx.+100 objects) problems. The bayersian estimates were also very low, it wasnt just the fit.
